src/libs/kata-types/src/container.rs

```diff
@@ -136,6 +137,39 @@ impl State {
     }
 }
 
+/// Update annotations: Remove specified keys or add new key-value pairs.
```

Comment:
Now that the `allow_overwrite` option has been removed, is it worth mentioning in this doc that attempts to overwrite the annotation are ignored?